README.md

```diff
@@ -1,16 +1,48 @@
-# thelazyentry
 
-No time to format your time entries? We got you covered :)
+<h1 align="center">thelazyentry 🥱</h1>
 
-Frameworks: `VanillaJS`
+ 
+</h2>
+
+<h3 align="center"> No time to format your time entries? We got you covered :)</h3>
+<br>
+
+<div align="center">
+<img src="https://img.shields.io/badge/made%20by-Nikhil Joshua-blue.svg" >
+
+<img src="https://img.shields.io/badge/VanillaJS -41B883.svg">
+
+<img src="https://img.shields.io/github/stars/NikhilJoshua/thelazyentry.svg?style=flat&color=orange">
+
+<img src="https://img.shields.io/github/languages/top/NikhilJoshua/thelazyentry.svg?style=flat&color=informational">
+
+<img src="https://img.shields.io/github/issues/NikhilJoshua/thelazyentry.svg">
+
+<img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat">
+</div>
+<br>
 
 # Features
 - Formats Entries
 - Copies in plain text for task entry
 - Copies rich text data for comment description
 
-# How to contribute
+## Future 🎯
+
+See the [open issues](https://github.com/NikhilJoshua/thelazyentry/issues) for a list of proposed features (and known issues).
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+To contribute to project, follow these steps:
+
+1. Fork this repository.
+2. Create a branch: `git checkout -b <your_branch_name>`.
```

Comment:
Branch name ideally should be your-name/<feature or fix> What do you think @PiyushKhurana ?

Reply:
Yep I agree 👍

Reply:
Never mind, forking will show the name. Sorry my bad. :)